suzieq/cli/sqcmds/PathCmd.py

```diff
@@ -25,6 +25,7 @@ def __init__(
         src: str = "",
         dest: str = "",
         vrf: str = "",
+        query_str: str = ' ',
```

Comment:
Are you passing a string with an empty char on purpose or it is a mistake?

Reply:
All query_str have an blank char string. I don't recall why its so.

Reply:
Oh I remember why. One of the pandas' versions had an issue with the query-str if we didn't end with a blank. I don't know if this is still the case with the 1.3.4 version we're using.